index.js

```diff
@@ -345,6 +345,16 @@ class TestFairy {
 	static setFeedbackOptions(options) {
 		TestFairyBridge.setFeedbackOptions(options);
 	}
+
+	/**
+	 * Attach a file to the session timeline at current moment in time.
+	 * 
+	 * @param filename Name of the attached file. It must have a file extension.
+	 * @param content A utf-8 javascript string, can be empty.
+	 */
+	static attachFile(filename, content) {
```

Comment:
can we please have mime types? like in fs.writeFile?
right now we just guess the mime type based on extension. 
we can ignore the content-type until we have this available in the sdk, but let's have it in react's api

Reply:
Done